docker/Dockerfile.cuda-base

```diff
@@ -0,0 +1,18 @@
+# livepeer/cuda-base:11.7.1-cudnn8-devel-ubuntu20.04
+#
+# Base CUDA Develop image which contains CUDA SDK libs for the following architectures: linux amd64, linux arm64, windows amd64
+#
+# To build this image you need the following steps:
+# 1. Download NVIDIA CUDA SDK for ARM64, extract, and copy into cuda/arm64/usr/local/cuda/
+# 2. Download NVIDIA CUDA SDK for Windows AMD64, extract, and copy npp libs into npp_win/
+
+FROM nvidia/cuda:11.7.1-cudnn8-devel-ubuntu20.04
+
+COPY npp_win/lib/x64/* /usr/local/cuda/lib64/
+COPY npp_win/include/* /usr/local/cuda/include/
+
+RUN mkdir -p /usr/local/cuda_arm64/lib64/
+COPY cuda/arm64/usr/local/cuda/lib64/libnp* /usr/local/cuda_arm64/lib64/
+COPY cuda/arm64/usr/local/cuda/include /usr/local/cuda_arm64/include
+
+RUN ln -s /usr/local/cuda /usr/local/cuda_amd64
```

Comment:
should we be building/pushing this regularly as well? or does it need to be manual? (could add a comment in the Dockerfile if it needs to be manual always

Reply:
I'd keep this step manual because :
1. We never change the cuda version, so we don't need to rebuild this image
2. It'd be relatively difficult to automate, because of exe files